_helpers/import_application.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+import json
+
+import app
+from config import Config
+from db.queries import bulk_insert_application_record
+from flask import current_app
+from services import _SQS_QUEUE_URL
+from services import delete_messages
+from services import receive_messages
+
+
+def import_applications_from_queue():
+    batch_size = Config.SQS_BATCH_SIZE
+    visibility_time = Config.SQS_VISIBILITY_TIME
+    wait_time = Config.SQS_WAIT_TIME
+    application_messages = receive_messages(
+        _SQS_QUEUE_URL, batch_size, visibility_time, wait_time
+    )
+
+    application_json_list = []
+
+    if application_messages:
```

Comment:
Rather than a layer if indentation we could return early here?:

```python3
if not application_messages: return
```

Reply:
In terms of performance it doesn't matter as it's still returns early here (returning None). But removing layer of the indentation helps in code readability.  Done now